core/prelude-core.el

```diff
@@ -285,6 +285,24 @@ there's a region, all lines that region covers will be duplicated."
   (interactive)
   (byte-recompile-directory prelude-dir 0))
 
+(defun prelude-file-uid (filename)
+  "Return the UID of the FILENAME, normally as a string.
+
+See `file-attributes for more info.'"
+  (nth 2 (file-attributes filename)))
+
+(defun prelude-file-owned-by-user-p (filename)
+  "Return t if file FILENAME is owned by the currently logged in user."
+  (let ((uid (prelude-file-uid filename)))
+    (or (eq uid
```

Comment:
Ok, seems you're right. You should not, however, use `eq` - it's for symbol comparison. You should be using `equal` instead. Or you can check the type of the returned value and do `string=` for string and `=` otherwise.